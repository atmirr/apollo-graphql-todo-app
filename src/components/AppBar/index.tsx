import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import { AssignmentTurnedIn as TodoIcon } from '@material-ui/icons';
import { AppBar as MuiAppBar, Toolbar, Typography } from '@material-ui/core';

const useStyles = makeStyles(({ spacing }) => ({
  appBar: {
    alignItems: 'center',
  },
  icon: {
    marginRight: spacing(0.5),
  },
}));

function AppBar() {
  const classes = useStyles();
  return (
    <MuiAppBar position="static" className={classes.appBar}>
      <Toolbar>
        <TodoIcon className={classes.icon} />
        <Typography variant="h5">Todo List</Typography>
      </Toolbar>
    </MuiAppBar>
  );
}

export default AppBar;
