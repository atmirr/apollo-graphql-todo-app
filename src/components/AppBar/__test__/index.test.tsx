import React from 'react';
import '@testing-library/jest-dom';
import '@testing-library/jest-dom/extend-expect';
import { render } from '@testing-library/react';
import AppBar from '../index';

describe('<AppBar />', () => {
  it('should renders the component', () => {
    const { container } = render(<AppBar />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
