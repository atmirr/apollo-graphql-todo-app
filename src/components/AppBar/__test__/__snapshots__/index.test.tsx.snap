// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppBar /> should renders the component 1`] = `
<header
  class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary makeStyles-appBar-1 MuiPaper-elevation4"
>
  <div
    class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
  >
    <svg
      aria-hidden="true"
      class="MuiSvgIcon-root makeStyles-icon-2"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M19 3h-4.18C14.4 1.84 13.3 1 12 1c-1.3 0-2.4.84-2.82 2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm-7 0c.55 0 1 .45 1 1s-.45 1-1 1-1-.45-1-1 .45-1 1-1zm-2 14l-4-4 1.41-1.41L10 14.17l6.59-6.59L18 9l-8 8z"
      />
    </svg>
    <h5
      class="MuiTypography-root MuiTypography-h5"
    >
      Todo List
    </h5>
  </div>
</header>
`;
